src/interfaces/sales.interface.ts

```diff
@@ -6,4 +6,10 @@ export interface SalesDataItem {
 
 export interface SalesOrders {
   annualSalesByMonth: SalesDataItem[]
+}
+
+export interface currentMonthDetails {
+  currentMonthRevenue: number;
+  ordersOpen: number;
+  salesMonthClosed: number;
 }
```

Comment:
same here

Reply:
a block in javascript does not use semicolon